test-suite/test-squid-conf.sh

```diff
@@ -99,6 +94,12 @@ then
                 exit 0;
             fi
 
+            if ! grep -q "# *define *\b$defineName\b $defineValue" $autoconfHeader
+            then
+                echo "$here: WARNING: Skipping $configFile test because $defineName is not $defineValue in $autoconfHeader";
```

Comment:
Please note that we want to keep the second parameter (i.e. `$p2` and `$defineValue`) optional (rather than require it in all use cases).